main.py

```diff
@@ -2,6 +2,36 @@
 
 from django.db.models import QuerySet
 
+from db.models import Genre, Actor
+
 
 def main() -> QuerySet:
-    pass
+
+    genres_list_to_create = ["Western", "Action", "Dramma"]
+
+    for genre in genres_list_to_create:
+        Genre.objects.create(name=genre,)
+
+    actors_list_to_create = [("George", "Klooney"), ("Kianu", "Reaves"),
```

Comment:
assigning variable names should be at the start of function
by that you'll improve code readability and ensure that variables are defined and initialized in a logical order, making it easier to understand and maintain the code.